server/vulnerabilities/cpe.go

```diff
@@ -139,6 +139,20 @@ func cleanAppName(appName string) string {
 
 var onlyAlphaNumeric = regexp.MustCompile("[^a-zA-Z0-9]+")
 
+// upstreamVersionForUbuntu disassembles the version string
+// used by Ubuntu packages.
+//
+// Sample version: 2.31.1-0.4ubuntu3.7, this method would return the
+// version of the package (aka upstream version), 2.31.1.
+func upstreamVersionForUbuntu(software *fleet.Software) string {
```

Comment:
This should be moved to the cpe.sqlite generation if possible. But probably will be left for @michalnicp when he adds ubuntu there.